test/contracts/exec-same-ctx-parent/exec-same-ctx-parent.c

```diff
@@ -63,20 +65,68 @@ void parentFunctionChildCall() {
 			(byte*)executeArgumentsLengths,
 			executeArgumentsData
 	);
+
+	// TODO assert that the storage changes made by the child are visible here
+	finishResult(result);
+}
+
+void parentFunctionChildCall_BigInts() {
+	bigInt intA = bigIntNew(84); 
+	bigInt intB = bigIntNew(96);
+	bigInt intC = bigIntNew(1024);
+
+	byte argumentSize = sizeof(bigInt);
+
+	// All SmartContracts expect their integer arguments in Big Endian form, so
+	// we need to reverse them (we're in Little Endian here) in order to pass
+	// them to the childSC.
+	bigInt arguments[] = {
+		reverseU32(intA),
```

Comment:
These would be useful as helpers, yes, but only when called explicitly, not automatically by the Rust framework. 

`reverseU32()` is only needed here because the arguments passed to the child SC are known in advance to be integers. The decision to convert back to big-endian cannot be made automatically by the framework.